src/OrchardCore.Modules/OrchardCore.Users/ConfigureCookieAuthenticationOptions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+using OrchardCore.Settings;
+using OrchardCore.Users.Controllers;
+
+namespace OrchardCore.Users
+{
+    public class ConfigureCookieAuthenticationOptions : IConfigureNamedOptions<CookieAuthenticationOptions>
+    {
+        private readonly IServiceScopeFactory _serviceScopeFactory;
+
+        public ConfigureCookieAuthenticationOptions(IServiceScopeFactory serviceScopeFactory)
+        {
+            _serviceScopeFactory = serviceScopeFactory;
+        }
+
+        public void Configure(CookieAuthenticationOptions options)
+        {
+            Configure(null, options);
+        }
+
+        public void Configure(string name, CookieAuthenticationOptions options)
+        {
```

Comment:
> 
> 
> Anyway, seems better to use an `IConfigureOptions`, not an `IConfigureNamedOptions` because here the `name` parameter is never used.

It won't be better, .net core has a bug that does not run `IConfigureOptions` in this case